sdk/storage/storage-file/BreakingChanges.md

```diff
@@ -1,6 +1,12 @@
 # Breaking Changes
 
-2019.07 Version 12.0.0-preview.1
+### 2019.08
+
+- [Breaking] Aborter class is no longer exposed from the package. You are expected to use
+  the package [@azure/abort-controller](https://www.npmjs.com/package/@azure/abort-controller) to pass an abort signal to any of the async operations.
+  [`AbortController.timeout(<milliseconds>)` can be utitlized as an abort signal.]
```

Comment:
Why are there brackets around this line? I don't think they are necessary.

Reply:
Removed